build/yaml/webhook/nsx-operator.yaml

```diff
@@ -0,0 +1,510 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: nsx-operator
```

Comment:
there might be other webhooks added, could we change name to be more specific to this one?